kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -74,6 +74,11 @@
     //
     // --- Kafka on Pulsar Broker configuration ---
     //
+    @FieldContext(
+            category = CATEGORY_KOP,
+            doc = "The number of threads used to respond to the response."
+    )
+    private int sendKafkaResponseThreads = Runtime.getRuntime().availableProcessors();
```

Comment:
The default value of sendKafkaResponseThreads may not need `Runtime.getRuntime().availableProcessors();` so many threads.

Reply:
Give default value = 5 ? Because this is a common thread, the specific value depends on the cluster pressure.

Reply:
I modified the default value to 4  @hangc0276